_posts/2020-02-03-release-1.10.0.md

```diff
@@ -0,0 +1,187 @@
+
+Flink 1.10 also marks the completion of the [Blink integration](https://flink.apache.org/news/2019/08/22/release-1.9.0.html#preview-of-the-new-blink-sql-query-processor), hardening streaming SQL and bringing mature batch processing to Flink with production-ready Hive integration and TPC-DS coverage. This blog post describes all major new features and improvements, important changes to be aware of and what to expect moving forward.
+
+{% toc %}
+
+The binary distribution and source artifacts are now available on the updated [Downloads page]({{ site.baseurl }}/downloads.html) of the Flink website. For more details, check the complete [release changelog]() and the [updated documentation]({{ site.DOCS_BASE_URL }}flink-docs-release-1.10/). We encourage you to download the release and share your feedback with the community through the [Flink mailing lists](https://flink.apache.org/community.html#mailing-lists) or [JIRA](https://issues.apache.org/jira/projects/FLINK/summary).
+
+
+## New Features and Improvements
+
+
+### Improved Memory Management and Configuration
+
+The current `TaskExecutor` memory configuration in Flink has some shortcomings that make it hard to reason about or optimize resource utilization, such as: 
+
+* Different configuration models for memory footprint in Streaming and Batch execution; 
+
+* Complex and user-dependent configuration of off-heap state backends (i.e. RocksDB) in Streaming execution.
+
+To make memory options more explicit and intuitive to users, Flink 1.10 introduced significant changes to the `TaskExecutor` memory model and configuration logic ([FLIP-49](https://cwiki.apache.org/confluence/display/FLINK/FLIP-49%3A+Unified+Memory+Configuration+for+TaskExecutors)). These changes make Flink more adaptable to all kinds of deployment environments (e.g. Kubernetes, Yarn, Mesos), giving users strict control over its memory consumption.
+
+**Out-of-the-box Managed Memory Configuration**
+
+Managed memory was extended to also account for memory usage of state backends. This means that there may be scenarios where jobs in the same cluster require different types of memory (e.g. a streaming job with `MemoryStateBackend` and another streaming job with `RocksDBStateBackend`). To enable these scenarios and allow users to switch between Streaming and Batch execution without having to modify cluster configurations, managed memory is now always off-heap.
+
+**Simplified RocksDB Configuration**
+
+Configuring an off-heap state backend like RocksDB used to involve a good deal of manual tuning, like decreasing the JVM heap size or setting Flink to use off-heap memory. This can now be achieved through Flink's out-of-box configuration, and adjusting the memory budget for `RocksDBStateBackend` is as simple as resizing the managed memory size. 
```

Comment:
```suggestion
Configuring an off-heap state backend like RocksDB used to involve a good deal of manual tuning, like decreasing the JVM heap size or setting Flink to use off-heap memory. This can now be achieved through Flink's out-of-box configuration, making adjusting the memory budget for `RocksDBStateBackend` is as simple as resizing the managed memory size. 
```